darts/dataprocessing/transformers/window_transformer.py

```diff
@@ -0,0 +1,150 @@
+from typing import Iterator, List, Optional, Sequence, Tuple, Union
+
+from darts.dataprocessing.transformers import BaseDataTransformer
+from darts.logging import get_logger
+from darts.timeseries import TimeSeries
+from darts.utils.utils import series2seq
+
+logger = get_logger(__name__)
+
+
+class ForecastingWindowTransformer(BaseDataTransformer):
```

Comment:
Since it's not necessarily for forecasting only, could we rename it to `WindowTransformer` ?